clients/p4rt-ctl/p4rt-ctl.in

```diff
@@ -660,6 +662,26 @@ class P4RuntimeClient:
             pass
         return None
 
+    def get_packet_in(self, timeout=3):
+        msg = self.get_stream_packet("packet", timeout)
+        if msg is not None:
+            return msg.packet
+
+    def send_packet_out(self, payload):
+        packet_out = p4runtime_pb2.PacketOut()
+        packet_out.payload = payload
+        packet_out_req = p4runtime_pb2.StreamMessageRequest()
+        packet_out_req.packet.CopyFrom(packet_out)
+        self.stream_out_q.put(packet_out_req)
+
+    # Continuously poll for pkts from the server
+    def pktio_rx(self, tap_device):
+        print("Waiting for Rx packets...")
```

Comment:
Do we want to have a print statement inside a continuous poll function? This will spam and slow down

Reply:
Are you asking about` print("Waiting for Rx packets...")`? This line is outside the while loop which checks for packets continuously.